CONTRIBUTING.md

```diff
@@ -4,11 +4,11 @@ The Mapbender team welcomes contributions from all members - so you are welcome
 
 Third-party patches are essential for the preservation of high standards in [Mapbender].
 
-We simply cannot access the huge number of platforms and myriad configurations for running [Mapbender]. 
+We simply cannot access the huge number of platforms and myriad configurations which run [Mapbender]. 
 
 We want it as easy as possible to carry out changes to get the [modules] in your environment to run. 
 
-There are a few [guidelines][rules] that we need contributors to follow so that we can have a chance of keeping on top of things.
+Therefore, we provide a few [guidelines][rules] that every contributor has to follow, in order that we have a chance of keeping on top of things.
```

Comment:
I would recommend:
`Therefore, we provide [contribution-guidelines][rules], in order to have a chance of keeping on top of things.`

What is the meaning of `keeping on top of things`? Maybe we can find a better wording.